### PR TITLE
Allow embeds to be used in extensions.

### PR DESCRIPTION
**Pull request by bryans99**
Demo embed in an extension

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "node": ">=10"
   },
   "dependencies": {
-    "@looker/components": "0.7.2",
+    "@looker/components": "0.7.13",
+    "@looker/embed-sdk": "^1.0.0-beta.4",
```
Comment (whscullin) [inline package.json]: I guess we should bump the version.

File: src/components/Embed/Embed.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  ExtensionContextData,
+} from "@looker/extension-sdk-react"
+import { Button } from "@looker/components"
+
+export const Embed: React.FC<EmbedProps> = () => {
+  const [dashboard, setDashboard] = React.useState<LookerEmbedDashboard>(null)
+  const extensionContext = useContext<ExtensionContextData>(ExtensionContext)
+
+  const canceller = (event: any) => {
+    console.log(">>>>> in canceller")
```
Comment (whscullin) [inline src/components/Embed/Embed.tsx]: Not sure what the policy on `console.log` is here.

Comment (bryans99) [reply src/components/Embed/Embed.tsx]: Removing it. Forgot it was there

File: package.json (hunk 1 above)
Comment (whscullin) [reply package.json]: Does this pick up 1.0.0-beta.5? I haven't had much luck with beta point releases and semver.

Comment (whscullin) [reply package.json]: Yeah, this doesn't pick up beta.5. it needs to be `1.0.0-beta.5`. It seems neither ^ nor ~ pick up beta revisions.

Comment (bryans99) [reply package.json]: Thanks for pointing it out. I tried it myself and came to the same conclusion. Will push fix soon

Comment (bryans99) [reply package.json]: Fixed

**Pull request merged**